api/src/models/project-view.ts

```diff
@@ -4,13 +4,12 @@ import { ProjectUser } from '../repositories/project-participation-repository';
 import { SystemUser } from '../repositories/user-repository';
 
 export interface IProjectAdvancedFilters {
-  permit_number?: string;
   project_programs?: number[];
   start_date?: string;
   end_date?: string;
   keyword?: string;
   project_name?: string;
-  species?: number[];
+  species_tsns?: number[];
```

Comment:
nit: can we call this `itis_tsns`?

Not sure if we are 100% consistent, but for the most part we've been using `itis_*`

Reply:
Good call.